docs/en/integrations/data-ingestion/clickpipes/object-storage.md

```diff
@@ -44,6 +44,10 @@ You have familiarized yourself with the [ClickPipes intro](./index.md).
 
   ![Use and existing table](./images/cp_step4b.png)
 
+:::info
+You can also map [virtual columns](../../sql-reference/table-functions/s3#virtual-columns), like `_path` or `_size`, to fields.
```

Comment:
optional: we could unify the representation. see Kafka docs https://github.com/ClickHouse/clickhouse-docs/pull/2260